src/components/tl/ucp/tl_ucp.c

```diff
@@ -175,6 +175,32 @@ static ucs_config_field_t ucc_tl_ucp_context_config_table[] = {
      ucc_offsetof(ucc_tl_ucp_context_config_t, pre_reg_mem),
      UCC_CONFIG_TYPE_UINT},
 
+    {"SERVICE_WORKER", "0",
```

Comment:
@samnordmann @Sergei-Lebedev i've opened a PR against Sam's branch : https://github.com/samnordmann/ucc/pull/1. I think this is more convenient, and then we don't need to add these service variables to TL/UCP. wdyt?

Reply:
I think it is great. I merge your PR to my branch